tests/test_pipelines.py

```diff
@@ -72,7 +72,7 @@ def test_datasets(self):
                     "illumMaskedImage", "deferredChargeCalib",
                     # Everything else
                     "skyMap", "gaia_dr3_20230707", "gaia_dr2_20200414", "ps1_pv3_3pi_20170110",
-                    "goodSeeingCoadd", "pretrainedModelPackage",
+                    "goodSeeingCoadd", "pretrainedModelPackage", "regionTimeInfo"
```

Comment:
`regionTimeInfo` should not be a pipeline input, it's made by `getRegionTimeFromVisit`. Are you saying this test still doesn't pass without this change?

Reply:
Yes, when I remove this it fails with 
`tests/test_pipelines.py::PipelineDefintionsTestSuite::test_datasets - AssertionError: Got unexpected pure_inputs: ['regionTimeInfo']`

Reply:
Ok, I'll look into it myself. I didn't see anything obviously wrong on the PR.

Reply:
Ah, now I see -- `ApPipe` is fine, it's `Ephemerides.yaml` that's causing the problem. Which is fair, because it's in fact _not_ a self-contained pipeline. Moving `getRegionTimeFromVisit` into `Ephemerides` is not viable, because that task is also a prerequisite for `loadDiaCatalogs`.

Honestly, my preferred solution would be to merge `Ephemerides` into `ApPipe` -- I don't think having a separate file for this task adds anything, and it makes the logic of `ApPipe` itself harder to follow because _both_ its inputs and outputs are actually used by tasks that are directly in `ApPipe`. The alternative is to break up `test_datasets` so that it's actually testing `Ephemerides` separately from `ApPipe` and `ApPipeWithFakes`.